### PR TITLE
Fix an exception with an empty or malformed corpus last sync file (#1279).

### PR DESCRIPTION
**Pull request by Dor1s**

### CODE REVIEW

**Comment by Dor1s** [comment]
> You can also try catch the thing, if lets say the file was corrupted, that might be better than just "if last_sync_time"

`utils.read_data_from_file` seems to be doing that and returns `None` in such case. I can do `eval_data=False`, in that case it should be fair to say that the file is empty

**Comment by inferno-chromium** [comment]
> > You can also try catch the thing, if lets say the file was corrupted, that might be better than just "if last_sync_time"
> 
> `utils.read_data_from_file` seems to be doing that and returns `None` in such case. I can do `eval_data=False`, in that case it should be fair to say that the file is empty

I mean datetime.datetime.utcfromtimestamp can exception if the content is corrupt ?

**Comment by Dor1s** [comment]
Yes, this is what happens right now. Let me upload another patch to look at, since the tests seem to be broken anyway:

```
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 77
```

File: src/python/bot/tasks/fuzz_task.py [hunk 1]
```diff
@@ -419,6 +419,28 @@ def _track_testcase_run_result(fuzzer, job_type, new_crash_count,
   })
 
 
+def _last_sync_time(sync_file_path):
+  """Read and parse the last sync file for the GCS corpus."""
+  if not os.path.exists(sync_file_path):
+    return None
+
+  file_contents = utils.read_data_from_file(sync_file_path, eval_data=False)
```
Comment (oliverchang) [inline src/python/bot/tasks/fuzz_task.py]: nit: why not just let lines 432 onwards handle this case also?

Comment (Dor1s) [reply src/python/bot/tasks/fuzz_task.py]: My understanding is that empty file is ok (therefore just a warning), while corrupted means we have a problem and an error should be reported. My initial patch was to log a message like "either malformed or empty" and Abhishek suggested to catch the failure and separate these

**Pull request merged**